saml2/saml2-service-provider/src/main/java/org/springframework/security/saml2/provider/service/web/Saml2MetadataFilter.java

```diff
@@ -17,6 +17,8 @@
 package org.springframework.security.saml2.provider.service.web;
```

Comment:
In any files you change, will you please make sure that the copyright is updated to the current year?